tasks/wdl/ImputationBeagleTasks.wdl

```diff
@@ -42,36 +42,109 @@ task CalculateContigsToProcess {
 
 }
 
-task CountVariantsInChunks {
+task ExtractUniqueVariantIds {
   input {
     File vcf
     File vcf_index
```

Comment:
now that you are streaming in the file you dont need the index input but honestly not a big deal either way